spec/ics-3-connection-semantics/README.md

```diff
@@ -0,0 +1,143 @@
+---
+ics: 3
```

Comment:
I think this oversteps ICS 3 - Connection Semantics?

Reply:
It's the same spec, we'll merge that PR into this one or vice-versa.